pkg/cli/admin/release/mirror.go

```diff
@@ -817,7 +817,7 @@ func (o *MirrorOptions) Run() error {
 		}
 	} else if len(toList) > 0 {
 		if o.PrintImageContentInstructions {
-			if err := printImageContentInstructions(o.Out, o.From, toList, o.ReleaseImageSignatureToDir, repositories); err != nil {
+			if err := printImageDigestMirrorInstructions(o.Out, o.From, toList, o.ReleaseImageSignatureToDir, repositories); err != nil {
```

Comment:
What happens if user's client is 4.14 but server is 4.13?, we need to keep backwards compatibility with +1/-1 version skew and I think, that's why we should support both idms and icsp for at least 2 versions.

There would be a some way to understand which one server supports and we print a instruction message according to it.

Reply:
> There would be a some way to understand which one server supports and we print an instruction message according to it.

I see, will take a look to find the server version

Reply:
Instead of checking according to the server version, what about checking the IDMS from server?, wouldn't it be more generic approach?.

Reply:
hmm, it seems this command does not need to have a running cluster, so we can not check the idms object.

Reply:
The version of the release image is already extracted at line 518, I think we can parse this, if the version is 4.14 or above we can print out instructions for IDMS.
https://github.com/openshift/oc/blob/c6df0e998fd1fd1c54586e5b6e9e366a91916f20/pkg/cli/admin/release/mirror.go#L518